x/airdrop/genesis.go

```diff
@@ -0,0 +1,21 @@
+package airdrop
+
+import (
+	"github.com/ingenuity-build/quicksilver/x/airdrop/keeper"
+	"github.com/ingenuity-build/quicksilver/x/airdrop/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	k.SetParams(ctx, genState.Params)
```

Comment:
Will need to set ClaimRecords from `genState.Allocations` (or whatever we call it); but it'll literally be a case of:
1. iterate records in genState
2. sum values
3. check total value vs module pool balance (and error if it is too much)
4. transfer from module pool to per chain pool

Reply:
Also (for the case of future chain exports) we need to be able to support exporting the ZoneDrop records too.